code/modules/admin/verbs/adminhelp.dm

```diff
@@ -507,23 +507,39 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 	log_ahelp(id, "Rejected", "Rejected by [usr.key]", null, usr.ckey)
 	Close(silent = TRUE)
 
-//Resolve ticket with IC Issue message
-/datum/admin_help/proc/ICIssue(key_name = key_name_admin(usr))
+/// I feel like this is a really roundabout way to do what I want, so I'm open to suggestions on making it more efficient.
```

Comment:
what's wrong with just using the GLOB.adminreplies list if they're already indexed by titles

Reply:
Can't parse GLOB.adminreplies directly into a list as it can't handle the fact that it's X = Y.

Reply:
so

```
var/input = tgui_input_list(yada yada yada, GLOB.admin_replies
if(!input)
 return
input = GLOB.admin_replies[input]
```
doesn't work?

Reply:
discussed in discord